docs/guides/avoid_getting_blocked.mdx

```diff
@@ -0,0 +1,67 @@
+---
+id: avoid-getting-blocked
+title: Avoid getting blocked
+description: How to avoid getting blocked when scraping using crawlee-python.
+---
+
```

Comment:
Please add:

```mdx
import ApiLink from '@site/src/components/ApiLink';
```

and use it for all references of Crawlee components, like `PlaywrightCrawler`, `BrowserPool`, ...

Reply:
Thank you, will implement using this!